docs/content/how-to/system-tables.md

```diff
@@ -256,6 +256,25 @@ SELECT * FROM MyTable$manifests /*+ OPTIONS('scan.snapshot-id'='1') */;
 */
 ```
 
+### Aggregation Table
+
+You can query the historical aggregation of the table through aggregation table.
+
+```sql
+SELECT * FROM MyTable$aggregation;
+
+/*
++------------+-----------------+---------------------+---------+
+| field_name |            type | AggregationFunction | comment |
```

Comment:
field_name
field_type
function
function_options
comment

What is function_options? For example, fields.xx.min or fields.xx.max.

Reply:
done.